geonode/settings.py

```diff
@@ -736,12 +740,22 @@
     'GEOSERVER_LOCATION', 'http://localhost:8080/geoserver/'
 )
 
+GEOSERVER_PUBLIC_HOST = os.getenv(
+    'GEOSERVER_PUBLIC_HOST', SITE_HOST_NAME
+)
+
+GEOSERVER_PUBLIC_PORT = os.getenv(
+    'GEOSERVER_PUBLIC_PORT', 8080
+)
+
+_default_public_location = 'http://{}:{}/geoserver/'.format(GEOSERVER_PUBLIC_HOST, GEOSERVER_PUBLIC_PORT) if GEOSERVER_PUBLIC_PORT else 'http://{}/gs/'.format(GEOSERVER_PUBLIC_HOST)
```

Comment:
This must be `http://{}:{}/gs/` instead of `http://{}:{}/geoserver/`

Reply:
the result url is not found for default values ..  http://localhost:8080/gs
i think it should refere to geoserver url as the definition is for public port & host for geoserver meanwhile /gs is geonode url( act as a proxy for geoserver)

Reply:
Yes, because of the default GEOSERVER_PUBLIC_PORT. It should be 8000 instead of 8080.

The policies are not to pass through the GeoNode proxy.

Reply:
I have the impression that here we introduced a regression:

if `_default_public_location` ends always up with the form `http://{}:{}/gs/` then the default result of GEOSERVER_WEB_UI_LOCATION is wrong because neither django urls nor jetty/tomcat have such a location `/gs`

@afabiani @giohappy what do you think?

Reply:
GEOSERVER_WEB_UI_LOCATION should be a variable with it's own value

Reply:
Is it currently wrong indeed?

Reply:
@francbartoli not if we are going to use GeoNode proxy